pkg/adapter/adapter.go

```diff
@@ -126,6 +126,17 @@ func (adp *LocalAdapter) Convert(ctx context.Context, source string) error {
 	adp.content.GcMutex.RLock()
 	defer adp.content.GcMutex.RUnlock()
 	if _, err = adp.cvt.Convert(ctx, source, target, cacheRef); err != nil {
+		if errdefs.NeedsRetryWithOutCache(err) && cacheRef != "" {
+			logrus.Infof("retry convert without cache: %s", source)
+			// clear lru cache in content and convert agagin without cache
+			if err = adp.content.NewRemoteCache(cacheRef); err != nil {
+				return err
+			}
+			if _, err := adp.cvt.Convert(ctx, source, target, ""); err != nil {
```

Comment:
Can we clean the cache in the `Convert` func?

Reply:
maybe we can clear and new a cache in provider